### DISCUSSION TITLE
Modifying the image of Application Life Cycle hooks to match the Light Mode Page

### DISCUSSION THREAD

**Discussion by rsaz** [expressots/expresso-site-doc#47] [General]
### Guidelines

- [X] I agree to follow this project's Contributing Guidelines.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Existing Issues

N/A

### Current behavior

For dark mode the [Application Life Cycle hook](https://doc.expresso-ts.com/docs/overview/application#hooks-execution-order) is very visible as demonstrated below
![image](https://github.com/expressots/expresso-site-doc/assets/4088378/742ade81-a443-4af0-a573-8016eb9d11d7)

However, is not matching the light mode of the page as show below
![image](https://github.com/expressots/expresso-site-doc/assets/4088378/cc4b2394-1de9-415f-b694-57e7bfea827f)

The idea is to create the same image with the same content that matches the doc light mode

### Steps to reproduce

N/A

### Expected behavior

An image that show the same workflow, content matching the Light mode

### Package version

N/A

### Node.js version

N/A

### In which operating systems have you tested?

- [X] macOS
- [X] Windows
- [X] Linux

**Comment by dotarjun** [comment]
Hey can I give this a shot

**Comment by rsaz** [comment]
100%, go ahead @dotarjun

**Comment by rsaz** [comment]
> Hey can I give this a shot

Ticket assigned to you as well. Let us know about the status when you can. Regards

**Comment by rsaz** [comment]
Task no completed during the `Hacktoberfest` period
